src/index.js

```diff
@@ -59,7 +59,7 @@ const { context, GitHub } = require("@actions/github");
   } catch (error) {
     if (isPullRequest) {
       const token = core.getInput("github-token", { required: true });
-      const github = new GitHub(token);
+      const github = getOctokit(token);
       await github.issues.createComment({
```

Comment:
Did you mean to remove this line?

Reply:
fixed in [`7f252d0` (#56)](https://github.com/Financial-Times/origami-percy/pull/56/commits/7f252d0cd76056947afada78531ee11d3b232cbd) 👍